@ARMmbed/mbed-os-tools Please review

I am getting now after latest fetch, an error like this:

```
for mbed test command:

Test Case:
    Name: tests-usb_device-serial
    Path: .\TESTS\usb_device\serial
Traceback (most recent call last):
  File "python37-32\lib\site-packages\pkg_resources\__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "python37-32\lib\site-packages\pkg_resources\__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "python37-32\lib\site-packages\pkg_resources\__init__.py", line 791, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (mbed-os-tools 0.0.8 (c:\programs\python37-32\lib\site-packages), Requirement.parse('mbed-os-tools==0.0.6'), {'mbed-greentea'})
```

--- a/requirements.txt
+++ b/requirements.txt
@@ -14,7 +14,7 @@ Jinja2>=2.10.1,<2.11
 intelhex>=1.3,<=2.2.1
 mbed-ls>=1.5.1,<1.8
 mbed-host-tests>=1.4.4,<1.6
-mbed-greentea>=0.2.24,<1.7
+mbed-greentea>=0.2.24,<1.8
 beautifulsoup4>=4,<=4.6.3
 pyelftools>=0.24,<=0.25
 manifest-tool==1.4.8

